Add the initial metrics section overview

philipwalton opened the pull request:
Fixes #1568, fixes: #1575

This PR adds the initial Metrics section overview page as well as the content for the "User-centric metrics" doc as well as the "Custom metrics" doc.

In a follow-up PR I'll add content for each of the individual metrics docs (which includes updating/renaming some of our existing metrics docs).

_**Note:** I'm currently using a Lighthouse illustration as a placeholder for the Metrics section illustration. Once the new illustration is added, I'll update, but DO NOT MERGE until that happens._

kaycebasques commented on src/site/content/en/metrics/user-centric-metrics/index.md:
```diff
@@ -0,0 +1,199 @@
+
+But just because a metric is based on objective criteria and can be
+quantitatively measured, it doesn't necessarily mean those measurements are
+useful.
```
Are you still happy with this intro, given what you said in our chat?

> BTW, I'm realizing that I too often describe metrics as a means to measure performance, so I'm trying to go through and reframe certain parts (specifically in the custom metrics doc) to frame them as a tool to measure the user experience.

This intro heavily frames metrics as something related to performance

kaycebasques commented on src/site/content/en/metrics/user-centric-metrics/index.md:
```diff
@@ -0,0 +1,199 @@
+Over the past few years, members of the Chrome team—in collaboration with the
+[W3C Web Performance Working Group](https://www.w3.org/webperf/)—have been
+working to standardize a set of new APIs and metrics that more accurately
+measure how users experience the performance of a web page.
```
Another instance of defining metrics as measuring performance

kaycebasques commented on src/site/content/en/metrics/user-centric-metrics/index.md:
```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+title: User-centric metrics
```
Another option might be to rename this `Performance metrics` (the directory name should change too if you rename the title)

kaycebasques commented on src/site/content/en/metrics/custom-metrics/index.md:
```diff
@@ -0,0 +1,377 @@
+calculating the delta between each frame. If the delta is significantly longer
+than the display's framerate, you can report that as a long task. Such hacks are
+not recommended, though, because they actually affect performance themselves
+(e.g. drain battery).
```
When we're happy with all the content I'll do a copyedit to remove this `e.g.` and the others I didn't catch. Leave this comment unresolved for me.

philipwalton commented:
Taking another look, I agree that it's not realistic (given the current content) to remove the performance aspect and frame it entirely around user experience.

I've updated the title to add "Performance" back in, and I've updated the URLs as well.

I've also added the "metrics" illustration, so I'm removing the DO NOT MERGE label, as this can be merged as soon as the content is ready.
